hrui/css/hrui.css

```diff
@@ -5,3 +5,33 @@
 #crm-main-content-wrapper #contactDetails #govID span {
     margin-right: 1%;
 }
+
+#crm-container .crm-contact-page {
```

Comment:
If all this rules are for the same purpose (making the tabs vertical), maybe it would be better to write a couple line of comments before them explaining what you're doing here